### DISCUSSION TITLE
Link between PLUMED and i-PI whit deepMD-kit

### DISCUSSION THREAD

**Discussion by mcourtiel** [deepmodeling/deepmd-kit#3979] [Q&A]
Hello everyone,

I am trying to perform path-integral MD simulations using i-PI with a DeepMD potential and a bias created with PLUMED.

The versions I am using are:

    DeepMD version: 2.2.10
    PLUMED version: 2.8 (the one inside the DeepMD environment)
    i-PI version: [unknown, but it usually works well without PLUMED]

However, I am encountering the following error message:

The error message is the following one : 
  File "/PATH/TO/IPI/ipi/engine/forcefields.py", line 688, in __init__
    raise ImportError(
ImportError: Cannot find plumed libraries to link to a FFPlumed object/

I would appreciate any assistance in resolving this issue.
I think I might be missing something in the configuration or linking process.

Thank you in advance for your help!

**Comment by njzjz** [comment]
It seems ipi needs a Python interface of plumed, which is available in [`py-plumed`](https://github.com/conda-forge/py-plumed-feedstock)
